faiss/python/setup.py

```diff
@@ -16,17 +16,29 @@
 shutil.copyfile("__init__.py", "faiss/__init__.py")
 shutil.copyfile("loader.py", "faiss/loader.py")
 shutil.copyfile("swigfaiss.py", "faiss/swigfaiss.py")
+if os.path.exists("swigfaiss_avx2.py"):
+    shutil.copyfile("swigfaiss_avx2.py", "faiss/swigfaiss_avx2.py")
```

Comment:
Related to the above: it seems CMake's SWIG target generates `swigfaiss.py` regardless of whether it is an AVX2 build or not (and hence never generates anything called `swigfaiss_avx2.py`). Long term fix will be to introduce separate targets for `libfaiss`/`libfaiss_avx2` (and `swigfaiss`/`swigfaiss_avx2`), but for now I think we can just remove this `shutil.copyfile("swigfaiss_avx2.py", "faiss/swigfaiss_avx2.py")` as it is never executed.

Reply:
That's good.